### ISSUE TITLE
Formatter function dependencies not included

### ISSUE THREAD

**Issue by VitaliiK91** [highcharts/highcharts-react-native#79]
When function passed to formatter calls other functions (i.e. lodash or any other library) or functions defined outside of the scope of formatter function, it throws `Can't find variable` exception. This seems to be due to outer functions or libraries don't make it to the WebView where charts are rendered. How can we bypass it?
Example of not working formatter funtion:
```javascript
const sum = (a, b) => a + b;

function formatter() {
  return sum(this.value, 1);
}
```
will throw `Can't find variable add`

**Comment by sebastianbochan** [comment]
The problem here is that we use webview and we inject the javascript there. When you declare a variable in the app, the reference does not exist.

Im thinking how it should work, because the webview is really limited.

**Comment by VitaliiK91** [comment]
@sebastianbochan Yes, this is definitely why it is happening. The only solutions I see is to declare all the variables within the formatter function body or inject all dependencies in the web view, although both solutions sound suboptimal. Do you have any suggestions? Thanks

**Comment by sebastianbochan** [comment]
Do you mean to add variables + values from RN to webview by injectJavascript? It sounds really strange, especially that RN variables can change, but communication between layers is limited. Correct me if Im wrong.

**Comment by VitaliiK91** [comment]
@sebastianbochan For example I have a system wide value formatting engine, I already use to format values and input on RN side, and I would like to use same engine in chart formatters functions. The only way I found so far is to copy over and inject the entire engine with all dependencies in the web view.

**Comment by sebastianbochan** [comment]
@VitaliiK91,
Is it possible to paste a piece of example of it here? I would like to check this formatters.

**Comment by chandlervdw** [comment]
It feels like the issue resides in this line: https://github.com/highcharts/highcharts-react-native/blob/master/dist/src/HighchartsReactNative.js#L98

I'm very confused as to why passing a string works but passing an object doesn't.  I've tried several options – see @sebastianbochan's reference to my issue above.

**Comment by chandlervdw** [comment]
@VitaliiK91 ping ping. Can you help this keep [moving forward](https://github.com/highcharts/highcharts-react-native/issues/79#issuecomment-640385046)?

**Comment by gmonte** [comment]
any solution?

**Comment by aditya1711** [comment]
Is there a way to be able to access the Chart Object in these functions?
